samples/bluetooth/mesh/light_switch/src/model_handler.c

```diff
@@ -66,6 +71,12 @@ static void button_handler_cb(uint32_t pressed, uint32_t changed)
 		return;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_MESH_LOW_POWER) && (pressed & BIT(3))) {
+		bt_mesh_gatt_proxy_set(BT_MESH_FEATURE_ENABLED);
```

Comment:
This is wrong. LPN should start proxy service and should not enable GATT proxy at all. That means, either `bt_mesh_subnet_node_id_set()` or `bt_mesh_proxy_identity_enable()` API should be used. I would recommend `bt_mesh_proxy_identity_enable()`.

Reply:
![image](https://user-images.githubusercontent.com/35562897/196686013-47a47fa2-884a-46e7-a193-d8e93723abfa.png)

Reply:
The `bt_mesh_proxy_identity_enable` is conditionaly compiled if `CONFIG_BT_MESH_GATT_PROXY` is enabled

Reply:
It is possible to do this, but then you need to initialize the application by running `bt_mesh_gatt_proxy_set(BT_MESH_FEATURE_DISABLED)`

Reply:
However: Running node ID for 10 seconds consumes considerably more energy than running net id for 30.

Node ID:
![image](https://user-images.githubusercontent.com/35562897/196692422-1baad9db-8447-43ec-9c52-84117e85ecb0.png)

Net ID:
![image](https://user-images.githubusercontent.com/35562897/196692637-7db72bc9-0c7a-4f94-807a-1f4ddce72d11.png)

Is there a config option to decrease the frequency at which the Node ID advertises? If not I feel it is counterproductive to use Node ID for a low power sample when net id is the least powerconsuming alternative

Reply:
I get that advertising with Node ID is intended as the default choice when you want the client device to target a specific node as in this instance, but in its current state it feels a bit akward to set up, and it is not the best choise for low power consumption.

Reply:
> It is possible to do this, but then you need to initialize the application by running bt_mesh_gatt_proxy_set(BT_MESH_FEATURE_DISABLED)

Yes, maybe that's the alternative for now.

I see that there is confusion in the KConfig options related to Proxy Service and GATT Proxy. These are two independent features but they are somehow mixed up. This needs to be fixed seperately.

Reply:
Apologize for the late comment, but isn't setting `CONFIG_BT_MESH_GATT_PROXY_ENABLED=n` and calling `bt_mesh_proxy_identity_enable` by button press is enough? I tested this now and I see that:
1. I am able to connect and configure device after provisioning;
2. It advertises with Node Identity for 60 seconds;
3. When advertising for Node Identity stops, I don't see any connectable advertisements with Mesh Proxy Service issued by the device;
4. If I call `bt_mesh_proxy_identity_enable` on button press (without changing GATT Proxy state), I see advertisements with Node Identity again. I can connect to the device and configure it. After 60 seconds the connectable advs are off again.

Isn't it what we wanted to achieve?

Reply:
Hmm..., seems that I never discovered the `CONFIG_BT_MESH_GATT_PROXY_ENABLED` option. I will try it to verify that you are correct 👍 The power consumption is still larger for the Node Identity, but I guess that could be improved later.

Reply:
Yes, we can also reduce NODE_ID_TIMEOUT for now, and add Kconfig options to tweak adv intervals later.

Reply:
Yes, conceptually I like this better. Need to tweak the docs a bit, but should not require a lot of work.